app/controllers/carts_controller.rb

```diff
@@ -18,11 +21,19 @@ def destroy
     redirect_to carts_path
   end
 
-  def check_food_status(food, qty, food_id)
+  def check_food_status(food, qty, food_id, prep_time)
     unless food.status == "available"
       @ordered_foods.delete(food_id)
     else
       @total += (food.price * qty)
+      @prep_total += prep_time
+
     end
   end
+
+   def line_prep_total(qty, prep_time) 
+    added_time = (qty/7) * 10
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Surrounding space missing for operator `/`.